code/src/index.css

```diff
@@ -1,13 +1,350 @@
-  font-family: source-code-pro, Menlo, Monaco, Consolas, "Courier New",
-    monospace;
+  font-family: source-code-pro, Menlo, Monaco, Consolas, "Courier New", monospace;
+}
+header {
+  width: 100%;
+  display: block;
+  border-bottom: 1px solid #252525;
+  padding: 10px 20px 10px 20px;
+  position: fixed;
+  top: 0;
+  left: 0;
+  right: 0;
+  z-index: 2;
+  background: #222222e3;
+  backdrop-filter: blur(12px);
+}
+h1 {
+  font-size: 18px;
+}
+.albums {
+  padding: 15px;
+  padding-top: 60px;
+  display:inline-block;
+  display: flex;
+  flex-wrap: wrap;
+  align-items: flex-start;
+  align-content: flex-start;
+  height: 100vh;
+}
+.album {
+  display: block;
+  width: 20%;
+  box-sizing: border-box;
+  overflow: hidden;
+  background-color: #161616;
+  transform: scale(0.96, 0.96);
+  transition-timing-function: ease-in;
+  transition: 0.4s;
+  opacity: 0;
+  animation: fadeIn ease 1s;
+  animation-fill-mode: forwards;
+}
+.album:hover {
+  transform: scale(1, 1);
+}
+.artwork {
+  width: 100%;
+  display: inline-block;
+  position: relative;
+  border-radius: 8px;
+  overflow: hidden;
+}
+.artwork-image {
+  width: 100%;
+  z-index: 1;
+  display: block;
+}
+.description {
+  padding: 10px 10px 20px 10px;
+  display: inline-block;
+  width: 100%;
+  box-sizing: border-box;
+}
+.description a {
+  margin: 0 0 5px 0;
+  font-weight: 500;
+  white-space: nowrap;
+  overflow: hidden;
+  text-overflow: ellipsis;
+  color: #fff;
+  text-decoration: none;
+}
+/*If there are siblings ("+"), add " and " before the last item*/
+.artist-names span + span:last-child {
+		content: " & ";
+}
+.artist-names span + span:last-child:before {
+		content: " & ";
+}
+/*Add ", " before each item, after the first item. Exclude if it's the last item.*/
+.artist-names span:nth-of-type(n + 2):not(:last-child) {
+	content: ", ";
+}
+.artist-names span:nth-of-type(n + 2):not(:last-child)::before {
+	content: ", ";
+}
+.album-name a {
+  display: block;
+}
+.artist-names span {
+  color: #999999;
+}
+.description a:hover {
+  text-decoration: underline;
+}
+.artist-names span a:hover {
+  color: #fff;
+}
+.description a.description-album:visited {
+  color: #fff;
+}
+.description a.description-artist {
+  color: #999999;
+  margin: 0;
+  width: auto;
+}
+.album-actions {
+  position: absolute;
+  top: 0;
+  right: 0;
+  left: 0;
+  bottom: 0;
+  z-index: 2;
+  text-align: center;
+  opacity: 0;
+  background: #00000091;
+  transition: scale .5s ease;
+}
+.album-action {
+  width: 40px;
+  height: 40px;
+  display: inline-block;
+  position: relative;
+  top: calc(50% - 57px);
+  margin: 0 5px 0 5px;
+  fill: #fff;
+  transform: scale(0.85, 0.85);
+  transition-timing-function: ease-in;
+  transition: 0.4s;
+  z-index: 3;
+  cursor: pointer;
+}
+.album .album-action-play:hover {
+  transform: scale(1, 1);
+}
+.album .artwork:hover .album-actions {
+  opacity: 1;
+}
+.album-action-play {
+  width: 80px;
+  height: 80px;
+  top: calc(50% - 40px);
+}
+header select {
```

Comment:
I know you don't need to add classes to those elements (`header` and `select`), but it's considered as a good practice to style elements only by class names. Thanks to that, we achieve clear and flat structure (`class` styles are more important than `selector` styles and less important than `id` styles)